feature_apps_rpc/src/main/java/com/my/kizzy/feature_apps_rpc/AppsRpc.kt

```diff
@@ -64,12 +72,51 @@ fun AppsRPC(
         topBar = {
             LargeTopAppBar(
                 title = {
-                    Text(
-                        text = "Apps",
-                        style = MaterialTheme.typography.headlineLarge,
-                    )
+                    Column {
+                        Text(
+                            text = "Apps",
+                            style = MaterialTheme.typography.headlineLarge,
+                        )
+                    }
                 },
                 navigationIcon = { BackButton { onBackPressed() } },
+                actions = {
+                    if (isSearchBarVisible) {
+                        Column(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(10.dp)
+                        ) {
+                            TextField(
+                                value = searchText,
+                                onValueChange = { searchText = it },
+                                shape = RoundedCornerShape(topStart = 2.dp, topEnd = 2.dp),
+                                modifier = Modifier
+                                    .fillMaxWidth(),
+                                textStyle = MaterialTheme.typography.bodyLarge.copy(color = Color.Black),
```

Comment:
color shouldn't be set to black explicitly. It doesn't look good in dark theme

![Screenshot_20241008-133000_Kizzy](https://github.com/user-attachments/assets/b9da504d-84a4-4e34-a11e-9b0a4de03db0)

Reply:
That's part of what I was referring to in my comment yes
I'll fix it up when there's time 👍